staging/src/k8s.io/apiserver/pkg/util/openstack/types.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright 2021 The Arktos Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package openstack
+
+import (
+	"strconv"
+
+	v12 "k8s.io/api/apps/v1"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/json"
+)
+
+const (
+	LABEL_SELECTOR_NAME = "ln"
+	DEFAULT_NAMESPACE   = "kube-system"
```

Comment:
nit: I see kube-system floating around in a lot of places. Consider consolidate into a single shared constant.

Reply:
at least at this package, they are all replaced with the constant defined in here, expect the UT test input which is expected to use the kube-system string literally. 

any other place you refer to ?

Reply:
search kube-system in source folder, it appears everywhere

Reply:
./pkg/apis/core/types.go:       NamespaceSystem = "kube-system"

Reply:
i see what you have. those are existing type related. and this happens almost everyone in the k8s code base.
this const here defined is ONLY for this openstack package. so i will resolve.

Reply:
fixed to reuse the built-in const